app/core/speaker_diarization.py

```
@@ -50,9 +60,31 @@ def __init__(self,
+                print("✅ WAV 파일 감지, 직접 처리")
+                try:
+                    # librosa로 직접 로드
+                    y, sr = librosa.load(audio_path, sr=target_sr, mono=True)
+                    
+                    # 최종 WAV 파일로 저장
+                    with tempfile.NamedTemporaryFile(suffix="_final.wav", delete=False) as tmp_file:
+                        sf.write(tmp_file.name, y, target_sr)
+                        print(f"✅ WAV 전처리 완료: {tmp_file.name}")
+                        return tmp_file.name
+                        
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)